.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,21 @@
+name: CI-UnitTests
```

Comment:
I'm in strong favour of this file being renamed to `test` which feels like a standard